.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,24 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea for the nf-core website
```

Comment:
Should be about the pipeline, but it's an error in the TEMPLATE

Reply:
resolved: ee09cbc